pkg/commands/app.go

```diff
@@ -1226,6 +1228,89 @@ func NewCleanCommand(globalFlags *flag.GlobalFlagGroup) *cobra.Command {
 	return cmd
 }
 
+func NewVEXCommand(globalFlags *flag.GlobalFlagGroup) *cobra.Command {
+	vexFlags := &flag.Flags{
+		GlobalFlagGroup: globalFlags,
+	}
+	var vexOptions flag.Options
+	cmd := &cobra.Command{
+		Use:           "vex subcommand",
+		Aliases:       []string{"p"},
+		GroupID:       groupManagement,
+		Short:         "[EXPERIMENTAL] VEX utilities",
+		SilenceErrors: true,
+		SilenceUsage:  true,
+		PersistentPreRunE: func(cmd *cobra.Command, args []string) (err error) {
+			cmd.SetContext(log.WithContextPrefix(cmd.Context(), "vex"))
+
+			vexOptions, err = vexFlags.ToOptions(args)
+			if err != nil {
+				return err
+			}
+			return nil
+		},
+	}
+
+	repoCmd := &cobra.Command{
+		Use:           "repo subcommand",
+		Short:         "Manage VEX repositories",
+		SilenceErrors: true,
+		SilenceUsage:  true,
+		Example: `  # Initialize the configuration file
+  $ trivy vex repo init
+`,
+	}
+
+	repoCmd.AddCommand(
+		&cobra.Command{
+			Use:                   "init",
```

Comment:
Perhaps we need to add flag to disable vex repositories (remove `repository.yaml` or disable all repos).

At a minimum we should write in docs how to stop using vex repos

Reply:
It's activated only when `--vex repo` is passed. They can just remove the flag to stop using VEX repos.

Reply:
exactly! I didn't think about this.